archlinux/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+# Maintainer: Danil Osherov <shindo@yandex-team.ru>
+
+pkgname=blackhole
+pkgdesc="Yet another C++ logging library."
+pkgver=0.2.3
+pkgrel=1
+arch=('x86_64')
+url=('https://github.com/3Hren/blackhole')
+license=('LGPL3')
```

Comment:
I used PKGBUILD from some other project as a base for this one. So, sure, I will change it.